app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -242,6 +242,41 @@ class StoryFragmentTest {
     }
   }
 
+  @Test
+  fun testStoryFragment_changeConfiguration_chapterMissingPrerequisiteIsShownCorrectly() {
+    launch<StoryActivity>(createStoryActivityIntent()).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
+      onView(allOf(withId(R.id.story_chapter_list))).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          2
+        )
+      )
+      onView(atPositionOnView(R.id.story_chapter_list, 2, R.id.chapter_summary)).check(
+        matches(
+          withText("Complete Chapter 1: What is a Fraction? to unlock this chapter.")
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testStoryFragment_chapterMissingPrerequisiteIsShownCorrectly() {
```

Comment:
Small nit, but we usually put the orientation changed test after the non-orientation changed test since it's "building upon" previous assumptions verified by earlier tests.

Reply:
Ok, I moved it.